src/Arguments/Kusto/DatabaseListArguments.cs

```diff
@@ -3,4 +3,4 @@
 
 namespace AzureMcp.Arguments.Kusto;
 
-public class DatabaseListArguments : BaseClusterArguments;
```

Comment:
that's in purpose. 
BaseDatabaseArguments include DatabaseName while DatabaseList is in cluster scope.

Reply:
Indeed. thanks.